### PR TITLE
Add a tidbit on how to sponsor the virtual meetup

### PR DESCRIPTION
**Pull request by thejcannon**

### CODE REVIEW

File: docs/sponsors.md [hunk 1]
```diff
@@ -2,5 +2,7 @@
 
 Sponsorship for the PyTexas Virtual Meetup's been wrangled up by these fine organizations below. We're as grateful as a longhorn in spring grass! A big Texas-sized THANK YOU to each one of 'em.
 
+(Want to see your company or organization on this list? Send a warm howdy to sponsorship@pytexas.org)
```
Comment (thejcannon) [inline docs/sponsors.md:5]: I debated on also saying what sponsorship entails (e.g. a door prize). Thoughts on including that here?

File: docs/sponsors.md [hunk 2]
```diff
@@ -2,5 +2,7 @@
...
+
 ## JetBrains
-![JetBrains Logo](assets/images/jetbrains_logo.png){align=center width="300"}
\ No newline at end of file
+![JetBrains Logo](assets/images/jetbrains_logo.png){align=center width="300"}
```
Comment (thejcannon) [inline docs/sponsors.md:8]: I just used the GitHub Web UI editor, so I can't even say "oh looks like my editor made this change".

WTF

Comment (toonarmycaptain) [reply docs/sponsors.md:8]: Well, you can say that...

File: docs/sponsors.md (hunk 1 above)
Comment (toonarmycaptain) [reply docs/sponsors.md:5]: Could it not also entail helping out with primo-discord?

Comment (thejcannon) [reply docs/sponsors.md:5]: Feel free to suggest some wording, otherwise :man_shrugging:

Comment (MasonEgger) [reply docs/sponsors.md:5]: I think it's good enough. I'll take whatever they want to give. If someone wanted to make a 1 pager for Meetup sponsorship and link to it here I'd be fine with that.

**Review by MasonEgger — APPROVED** [review]
LGTM

**Pull request merged**